index.bs

```diff
@@ -712,44 +731,47 @@ Processing model {#processing}
 
 <h4 id="monkeypatch-html-windoworworkerglobalscope"><code>WindowOrWorkerGlobalScope</code></h4>
 
-Each [=user agent=] has an associated <dfn for="user agent">brands</dfn>, which is a [=/list=] created by running
-[=create brands=].
-
-Every {{WindowOrWorkerGlobalScope}} object has an associated <dfn for="WindowOrWorkerGlobalScope">brands frozen array</dfn>,
-which is a <code><a interface>FrozenArray</a>&lt;<a dictionary>NavigatorUABrandVersion</a>></code>. It is initially the
-result of [=create a frozen array|creating a frozen array=] from the [=user agent=]'s [=brands=].
+Each [=user agent=] has an associated <dfn for="user agent">brands</dfn>, which is a [=/list=]
+created by running [=create brands=] with [=user agent/significant version=].
 
-<h4 id="create-ua-list-section">Create brands</h4>
+Every {{WindowOrWorkerGlobalScope}} object has an associated
+<dfn for="WindowOrWorkerGlobalScope">brands frozen array</dfn>, which is a
+<code><a interface>FrozenArray</a>&lt;<a dictionary>NavigatorUABrandVersion</a>></code>. It is
+initially the result of [=create a frozen array|creating a frozen array=] from the [=user agent=]'s
+[=brands=].
 
-When asked to run the <dfn>create brands</dfn> algorithm, the [=user agent=] MUST run the following steps:
-1. Let |list| be a [=/list=].
+Additionally, every {{WindowOrWorkerGlobalScope}} object has an associated
+<dfn for="WindowOrWorkerGlobalScope">version list frozen array</dfn>, which is a
```

Comment:
Nit: "full version list frozen array" to make it more obvious?

Reply:
thanks - missed that one :)